cmd/metro2/main.go

```diff
@@ -157,39 +160,47 @@ var rootCmd = &cobra.Command{
 	Short: "",
 	Long:  "",
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-		if inputFile == "" {
-			path, err := os.Getwd()
-			if err != nil {
-				log.Println(err)
-			}
-			inputFile = path + "/" + "metro.json"
-		}
-		_, err := os.Stat(inputFile)
-		if os.IsNotExist(err) {
-			return errors.New("invalid input file")
-		}
-		f, err := os.Open(inputFile)
-		if err != nil {
-			return err
-		}
-		rawData = utils.ReadFile(f)
-
+		isWeb := false
 		cmdNames := make([]string, 0)
 		getName := func(c *cobra.Command) {}
 		getName = func(c *cobra.Command) {
 			if c == nil {
 				return
 			}
 			cmdNames = append([]string{c.Name()}, cmdNames...)
+			if c.Name() == "web" {
+				isWeb = true
+			}
 			getName(c.Parent())
 		}
 		getName(cmd)
+
+		if !isWeb {
+			if inputFile == "" {
+				path, err := os.Getwd()
+				if err != nil {
+					log.Println(err)
+				}
+				inputFile = path + "/" + "metro.json"
```

Comment:
Can we use `filepath.Join(path, "metro.json")`? That allows windows and mac/linux paths to be joined.